big-o-notation/big-o.js

```diff
@@ -0,0 +1,44 @@
+/* exported uniqueQuadratic, uniqueLinear */
+
+function uniqueLinear(words) {
+  const seen = {};                  // 1 * n = O(n)
+  const unique = [];                // 1 * n = O(n)
+  for (
+    let i = 0;                      // 1 * n = O(n)
```

Comment:
The assignment part of the for loop only runs once, not after every iteration so this line would still be `1 * 1`